clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ServerStoreProxy.java

```diff
@@ -44,25 +51,45 @@ public String getCacheId() {
 
   @Override
   public Chain get(long key) {
-    // TODO: Implement ServerStoreProxyImpl.get
-    throw new UnsupportedOperationException("ServerStoreProxyImpl.get not implemented");
+    try {
+      EhcacheEntityResponse response = entity.invoke(EhcacheEntityMessage.getOperation(cacheId, key), false);
+      if (response != null && response.getType() == EhcacheEntityResponse.Type.GET_RESPONSE) {
+        return ((EhcacheEntityResponse.GetResponse)response).getChain();
+      }
```

Comment:
if the response is null then the get returns null. I think rather it should throw Exception since even for absent keys the caller will get an empty chain, not null